### DISCUSSION TITLE
Bounding box colors

### DISCUSSION THREAD

**Discussion by JordiGrasvi** [blakeblackshear/frigate#7595] [Ask A Question]
Hello,

What means the blue or yellow bounding box color?

Thanks

**Comment by NickM-27** [comment]
you'll need to show an example. Different objects have different color bounding boxes. 

There is also the thinner dark blue bounding box which means the object is not currently detected

**Reply by JordiGrasvi** [reply]
For example this 2 images
![Captura de pantalla 2023-08-28 a les 16 41 27](https://github.com/blakeblackshear/frigate/assets/67150569/0341ae00-455e-4b42-8d46-deee5edb974d)
![Captura de pantalla 2023-08-28 a les 16 41 18](https://github.com/blakeblackshear/frigate/assets/67150569/d353eecf-d972-41b7-b567-cf68c206f5cc)

**Reply by NickM-27** [reply]
Right, those are just the colors assigned to those types of objects

**Reply by JordiGrasvi** [reply]
Even if it is the same object, can the color be different?

![Captura de pantalla 2023-08-28 a les 17 00 03](https://github.com/blakeblackshear/frigate/assets/67150569/e72b92a6-80e4-461b-9d61-b57dfb8e1f3e)
![Captura de pantalla 2023-08-28 a les 16 39 17](https://github.com/blakeblackshear/frigate/assets/67150569/d9c500ab-0866-44df-b359-f3c0e31fc854)

**Reply by NickM-27** [reply]
Like I said earlier the dark blue thin box means the object is currently bot detected

**Reply by wfchan** [reply]
what meaning of whte & orange background of bounding box?
![Screenshot 2024-05-24 at 2 51 56 PM](https://github.com/blakeblackshear/frigate/assets/30888356/3235eac8-0004-4b9f-b6f4-e00afc00344c)

**Reply by NickM-27** [reply]
That's new in 0.14 and means the object is stationary
